packages/rhi-webgl/src/WebGLGraphicDevice.ts

```
@@ -545,6 +563,7 @@ export class WebGLGraphicDevice implements IHardwareRenderer {
     }
 
     this._contextAttributes = gl.getContextAttributes();
+    debugger;
```

Biome (1.9.4)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)